client/src/entrypoints/admin/page-editor.js

```diff
@@ -177,6 +178,16 @@ function InlinePanel(opts) {
 
       self.updateMoveButtonDisabledStates();
       self.updateAddButtonState();
+      initCollapsiblePanels(
+        document.querySelectorAll(
+          `#inline_child_${newChildPrefix} [data-panel-toggle]`,
+        ),
+      );
+      console.log(
```

Comment:
Should this log statement have been removed?

Reply:
Yup!